healme.lic

```diff
@@ -470,6 +473,17 @@ class HealMe
 
   # Generate a key to uniquely identify this wound. Good for map keys.
   def build_composite_key_for_wound_details(body_part, wound_location, wound_type)
+    # Get rid of extraneous whitespace.
+    body_part = body_part.strip.downcase
+    wound_location = wound_location.strip.downcase
+    wound_type = wound_type.strip.downcase
+    # The game may refer to internal skin wounds as nerves, but the body part is skin.
+    # Standardize on skin as the body part.
+    wound_location = 'skin' if wound_location =~ /^nerves?$/
+    # The wound type may be passed to us either as singular (scar or wound) or plural (scars or wounds).
+    # Standardize on the singular variant.
+    wound_type = wound_type.delete_suffix('s')
+    # Build the key!
     "#{body_part}|#{wound_location}|#{wound_type}".downcase
```

Comment:
is this last downcase extraneous now?

Reply:
> is this last downcase extraneous now?

100%. I was paranoid. Dare I take it out...?

Reply:
I mean, it's not hurting anything if it makes you feel better. :D